crates/starknet-devnet/tests/test_fork.rs

```diff
@@ -584,6 +581,6 @@ mod fork_tests {
             .call(contract_call, BlockId::Tag(BlockTag::Latest))
             .await
             .unwrap();
-        assert_eq!(second_fork_block_number, [FieldElement::from(4_u8)]); // origin block + declare + deploy
+        assert_eq!(second_fork_block_number, [FieldElement::from(5_u8)]); // origin block + declare + deploy
     }
 }
```

Comment:
Have you considered adding a genesis block test when forking? To assert that a new block is created and that it's number is correct.

Reply:
Fixed in ebd8339e2576b5f0d94c108d360c95b15544bc58

Reply:
logic was added to test_forking_local_genesis_block()